Rakefile

```diff
@@ -1,16 +1,28 @@
+namespace :build do
+  desc 'Builds the Lottie package for iOS'
+  task :package do
+    sh 'xcodebuild build -scheme Lottie -destination generic/platform=iOS'
+  end
+
+  desc 'Builds the iOS Lottie Example app'
+  task :example do
+    sh 'xcodebuild build -scheme "Example (iOS)" -destination "platform=iOS Simulator,name=iPhone 8"'
```

Comment:
Perhaps as a follow-up, it would be nice to use `xcbeautify` here https://github.com/thii/xcbeautify which we should be able to install using Mint

Reply:
Great idea, will look in to this as a follow-up